app/assets/javascripts/charts.js

```diff
@@ -52,9 +52,17 @@ var Charts = {
     }
 
     if (data.xAxis) {
-      $.extend(data.xAxis.labels, {formatter: function(){
-        return this.value.stringify;
-      }});
+        $.extend(data.xAxis.labels, {formatter: function(){
+          var first_day = new Date(this.value)
+          var last_day = new Date(first_day.getFullYear(), first_day.getMonth()+1, 0);
+          var month = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
```

Comment:
This should be months instead of month for clarity.

Reply:
Please use the full Jira ticket number, i.e. OTWO-xxxx in your commits so we have an association between this work and the ticket.